src/_cffi_src/openssl/src/osrandom_engine.c

```diff
@@ -73,9 +85,24 @@ static int osrandom_init(ENGINE *e) {
 
 static int osrandom_rand_bytes(unsigned char *buffer, int size) {
     ssize_t n;
+    struct stat st;
+
+    if (fstat(urandom_cache.fd, &st)
```

Comment:
Please write this as `== -1`, also please put the `||` on the line above, not below.